### PR TITLE
fix lock file mounts

### PR DESCRIPTION
**Pull request by NyakudyaA**

### CODE REVIEW

**Comment by NyakudyaA** [comment]
For context read https://github.com/kartoza/docker-postgis/pull/309

File: README.md [hunk 1]
```diff
@@ -370,13 +370,36 @@ You can alternatively mount an extra  config file into the setting's folder i.e
...
 3. The service will delete your `DATADIR` directory and start reinitializing your data directory from scratch.
 
+### Lockfile
+
+ During container startup, some lockfile are generated which prevent reinitialisation of some
+ settings. These lockfile are by default stored in the `/settings` folder, but a user can control
+ where to store these files using the environment variable `CONF_LOCKFILE_DIR` Example
+
+ ```
+ -e /opt/conf_lockfiles \
```
Comment (dominicschaff) [inline README.md]: I think you are missing the env name here

File: scripts/env-data.sh [hunk 2]
```diff
@@ -6,10 +6,13 @@ DEFAULT_DATADIR="/var/lib/postgresql/${POSTGRES_MAJOR_VERSION}/main"
 # Commented for documentation. You can specify the location of
 # pg_wal directory/volume using the following environment variable:
 # POSTGRES_INITDB_WALDIR (default value is unset)
+DEFAULT_SCRIPTS_LOCKFILE_DIR='/docker-entrypoint.initdb.d'
```
Comment (dominicschaff) [inline scripts/env-data.sh]: Perhaps use double quotes here, so it's the same as the others

**Review by dominicschaff — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NyakudyaA) [reply README.md]: Fixed

File: scripts/env-data.sh (hunk 2 above)
Comment (NyakudyaA) [reply scripts/env-data.sh]: fixed

**Pull request merged**